DICUI/Utilities/Converters.cs

```diff
@@ -271,20 +271,26 @@ public static List<string> KnownSystemAndMediaTypeToParameters(KnownSystem? sys,
                 return null;
             }
 
+            //TODO: adjust parameters according to paranoid mode
```

Comment:
Add a paranoid one for DVD: `DICFlags.CMI`. That's the "Copyright Management Information" flag, and it is highly paranoid for DVDs. This counts for HD-DVD as well, since that's lumped under the DVD read flag.